dashboard/apps/prototype/management/commands/cache.py

```diff
@@ -25,6 +26,7 @@ def generate(self):
         """
         generate cache
         """
+        cache_tools.METHOD_CACHE_WRITE_ONLY = True
```

Comment:
i did start with your route. then i realised the `ignore_cached` flag will need to be passed down through several functions starting from `profile`, then `spendings_between` and finally `people_cost`, which makes a lot of methods cluttered.